centra_py_client/centra_py_client.py

```diff
@@ -75,3 +75,16 @@ def get_labels_ids(self, label_key, label_value):
         list_of_matching_label_ids = [x['id'] for x in list_of_matching_label_objects]
         self.logger.debug(f"Found {len(list_of_matching_label_ids)} matching labels")
         return list_of_matching_label_ids
+
+    def get_system_notifications(self):
+        return self.centra_session.json_query(
+            self.centra_session.urljoin_api('system-notifications')
```

Comment:
Did you test what happens when Centra is not really connected? I suspect this will throw an exception instead of returning None

Reply:
Good catch! Tested and added handling for that case as well - is it OK now?